rt_gene/src/rt_bene/estimate_blink_base.py

```diff
@@ -64,10 +64,10 @@ def load_img(self, img_path):
             print("ERROR: can't read " + img_path)
         return self.resize_img(img)
 
-    def predict(self, right_eyes, left_eyes):
+    def predict(self, left_eyes, right_eyes):
         with self.graph.as_default():
             tf.compat.v1.keras.backend.set_session(self.sess)
-            x = [np.array(right_eyes), np.array(left_eyes)]
+            x = [np.array(left_eyes), np.array(right_eyes)]
```

Comment:
Just below it can be seen that `model = Model(inputs=[right_input, left_input], outputs=merged)`, so here it should be `x = [np.array(right_eyes), np.array(left_eyes)]`.